docs/getting-started/first-abn.md

```diff
@@ -83,17 +79,24 @@ In another shell, run a script to generate load from multiple users:
 
 ## Deploy candidate
 
-Deploy the candidate version of the *backend* component, naming it `backend-candidate-1`.
+Release the application adding a second (candidate) version named `backend-candidate-1`:
```

Comment:
I think I still prefer the original wording. The new wording is less clear to me